docs/source/conf.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../../'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'circulation-admin'
```

Comment:
Does this control the URL on readthedocs? Something like `simplified-circulation` would probably make more sense.

Reply:
It does not. The name of the repo controls the url. I can change this to `simplified-circulation` for the title of the doc site.

Reply:
:+1: on changing the title